lib/libmuser.c

```diff
@@ -1207,8 +1212,13 @@ lm_ctx_create(const lm_dev_info_t *dev_info)
     lm_ctx->log_lvl = dev_info->log_lvl;
     lm_ctx->reset = dev_info->reset;
 
-    // Bounce the provided pci_info into the context.
-    memcpy(&lm_ctx->pci_info, &dev_info->pci_info, sizeof(lm_pci_info_t));
+    // Bounce info for IRQs and regions.
+    assert(sizeof(lm_ctx->pci.irq_count) == sizeof(dev_info->pci_info.irq_count));
```

Comment:
These should be static asserts closer to the definition of these structs. You only want the test to happen during compilation, not at runtime every time.

Reply:
I wanted to do it as you suggest however I don't know how to statically assert that a member of a struct is of a specific size.